Documentation/Troubleshooting/krew-plugin.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Krew Plugin
+---
+
+## Installation
+
+* Install [Krew](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)
+* Install Krew rook-ceph plugin
+  ```console
+    kubectl krew install rook-ceph
+  ```
+
+## Working
```

Comment:
Let's just have a single header to show an example, and no need for a subheader "Example"
```suggestion
## Ceph Status
```